ci/infra/aws/output.tf

```diff
@@ -1,11 +1,55 @@
-output "control_plane_public_ip" {
-  value = "${zipmap(aws_instance.control_plane.*.id, aws_instance.control_plane.*.public_ip)}"
+output "username" {
```

Comment:
As discovered, this is currently no-op and instance automatically uses `ec2-user`.

Reply:
Resolved in https://github.com/SUSE/skuba/pull/1325/commits/1317c3c94359df6664a37a5a81e01249ba8751f8